release/models/network-instance/openconfig-evpn.yang

```diff
@@ -1061,5 +1068,216 @@ module openconfig-evpn {
 
       uses evpn-vxlan-parameters-config;
     }
+
+    container endpoint-peers {
+      description
+        "Top level container for state information related to peer VXLAN Tunnel
+        Endpoints(VTEPs) learned by the local VTEP in the default network
+        instance";
+      config false;
+
+      list endpoint-peer {
+        key "peer-address";
+          description "List of VTEP peers and associated state information";
+
+        leaf peer-address {
+          type leafref {
+            path '../state/peer-address';
+          }
+          description "IP address for the VTEP peer";
+        }
+
+        container state {
+          config false;
+           description
+             "Container for state parameters related to this VTEP peer";
+             uses evpn-endpoint-peer-state;
+        }
+      }
+    }
+
+    container endpoint-vnis {
+      description
+        "Top level container for state information related to layer-2 and
+        layer-3 virtual network identifiers (L2/L3 VNIs) that are learned on
```

Comment:
You have a few different ways you reference L2VNI / L3VNI in the descriptions.

Can you do the following:

In line 1101/1102 indicate Layer 2 virtual network identifier (L2VNI) and layer 3 virtual network identifier (L3VNI) and then make a global replacement so that you are using the same like term throughout?